cmd/mindthegap/push/bundle/bundle.go

```diff
@@ -334,6 +167,217 @@ func withOnExistingTagMode(mode onExistingTagMode) pushOpt {
 	}
 }
 
+// PushConfig holds all configuration needed for pushing bundles.
+type PushConfig struct {
```

Comment:
With exporting it as config, the validation that comes with flags will not be available. 
wdyt adding some builder or validation to ensure that this config wont result into NPE

nit: wdyt moving this to `pkg` directory. Generally cmd packages should be internal to the repo.